src/main/java/org/folio/list/services/ListService.java

```diff
@@ -166,7 +167,15 @@ public Optional<ListRefreshDTO> performRefresh(UUID listId) {
         list.refreshStarted(getCurrentUser());
         TaskTimer timer = new TaskTimer();
         timer.start(TimedStage.TOTAL);
-        ListEntity savedList = timer.time(TimedStage.WRITE_START, () -> listRepository.save(list));
+        ListEntity savedList = timer.time(TimedStage.WRITE_START, () -> {
+          var tempSavedList = listRepository.save(list);
```

Comment:
Nit: specify `ListEntity`, not `var`

Reply:
Usually, I'd be right there with you. But in this particular instance, the type isn't relevant (nor is the name... Otherwise, the name would be pretty terrible)

Reply:
fair enough - just wanted to put it out there!

Reply:
It makes me wish there was some nice syntax for cases where you want to capture a value, run something, then return the value, and never need to create a temp variable. Like, if you could return a value from a try-with-context block, we could do some fairly simple shenanigans to make `TaskTimer` use that and do something like this (which is still clunky, but I like it better than needing a variable):
```java
ListEntity savedList = try (var task = timer.start(TimedStage.WRITE_START)) {
    listRepository.save(list);
} finally {
    entityManagerFlushService.flush();
}
```

This really really makes me wish we were using Kotlin, where it could be something like this (this probably isn't the best approach, but I bet it'd work):
```kotlin
val savedList: ListEntity = timer.time(TimedStage.WRITE_START) { 
    listRepository.save(list)
        .also {
            entityManagerFlushService.flush();
        }
}
```